presto-hive/src/main/java/com/facebook/presto/hive/HiveMetadata.java

```diff
@@ -2004,7 +2004,7 @@ private String createTableLayoutString(
     }
 
     @Override
-    public List<ConnectorTableLayoutResult> getTableLayouts(ConnectorSession session, ConnectorTableHandle tableHandle, Constraint<ColumnHandle> constraint, Optional<Set<ColumnHandle>> desiredColumns)
+    public List<ConnectorTableLayoutResult> getTableLayouts(ConnectorSession session, ConnectorTableHandle tableHandle, Constraint<ColumnHandle> constraint, Optional<Set<ColumnHandle>> requestedColumns)
```

Comment:
nit: keep the name the same as the interface

Reply:
will do